src/python/pants/releases/release.py

```diff
@@ -0,0 +1,107 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import hashlib
+import logging
+import os
+import urllib
+import xml.etree.ElementTree as ET
+from builtins import open
+from concurrent.futures import ThreadPoolExecutor
+
+import fire
+import requests
+
+from pants.net.http.fetcher import Fetcher
+from pants.util.dirutil import safe_mkdir
+
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+class Releaser(object):
+
+  OUTPUT_DELIMITER = '\t'
+
+  def list_prebuilt_wheels(self, binary_base_url, deploy_pants_wheels_path,
+                           deploy_3rdparty_wheels_path):
+    wheel_paths = []
+    for wheel_path in [deploy_pants_wheels_path, deploy_3rdparty_wheels_path]:
+      url = '{}/?prefix={}'.format(binary_base_url, wheel_path)
+      # can't figure out how not to get 400 with requests, so shell off to 'curl' instead.
+      resp = requests.get(url, allow_redirects=True, auth=None)
+      if resp.status_code != 200:
+        raise requests.exceptions.HTTPError(
+          'HTTP GET {} failed. You might want to remove '
+          'invalid entries in ~/.netrc (e.g. `default`)'.format(url))
```

Comment:
Instead of `You might want to remove`, I think it'd be better to say: `You may have`

Reply:
Adjusted.